src/Features/Core/Portable/SolutionCrawler/ISolutionCrawlerOptionsService.cs

```diff
@@ -0,0 +1,16 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Collections.Generic;
+using System.Text;
+using Microsoft.CodeAnalysis.Host;
+
+namespace Microsoft.CodeAnalysis.SolutionCrawler
+{
+    internal interface ISolutionCrawlerOptionsService : IWorkspaceService
```

Comment:
I introduced a new service as opposed to adding a new option on the existing services, so there was no risk this doesn't get deleted once we delete the solution crawler. Yes, I'm optimizing for deleting of this over anything else. :smile: